internal/business/persisted_operations/gcp_storage_loader.go

```diff
@@ -80,5 +106,17 @@ func (g *GcpStorageLoader) Load(ctx context.Context) error {
 		_ = reader.Close()
 	}
 
+	g.log.Info(fmt.Sprintf("Read %d manifest files from bucket", numberOfFilesProcessed))
+	reloadFilesGauge.WithLabelValues().Set(float64(numberOfFilesProcessed))
+
 	return errors.Join(errs...)
 }
+
+/*
+GCS exposes files with their names including directories (e.g. 'folder/filename') since we only
+care about the actual filename, we split the filename on '/' and pick the last part e.g. /folder/filename becomes filename
+*/
+func getFileName(attrs *storage.ObjectAttrs) string {
```

Comment:
This can also be achieved with stdlib via https://pkg.go.dev/path/filepath#Split